app/src/js/components/Logs/viewer.js

```diff
@@ -16,140 +18,160 @@ const noLogs = {
 };
 // const twoDays = 2 * 1000 * 60 * 60 * 24;
 
-const statusOptions = [
-  'All',
-  'Debug',
-  'Info',
-  'Warn',
-  'Error',
-  'Fatal',
-  'Trace'
-];
+// Use a Map to preserve order of entries.  Otherwise, when using a plain
+// object, during iteration over the keys (or entries), "numeric" keys are
+// yielded in numeric order, not in insertion order (Gah!).
```

Comment:
Can you point me the place where the keys were not ordered correctly before the changes?

Reply:
Ok.  I see the Log Level in granule page

Reply:
Right, this wasn't a problem before because there were no log level numbers before. This comment is specific to the new level numbers I added, which correspond to the change I made in the dropdown component to allow specifying option values that are different from the display values. In this case, this is because the API call wants numbers for the log levels, not names.